packages/web-pkg/src/composables/sse/useServerSentEvents.ts

```diff
@@ -24,7 +26,7 @@ export const useServerSentEvents = (options: ServerSentEventsOptions) => {
     }
   )
   const setupServerSentEvents = () => {
-    if (unref(retryCounter) >= 5) {
+    if (unref(retryCounter) >= 3) {
```

Comment:
Last nitpick: Please use a variable for the maximum amount of retries, since we're now using it here and down below 🙂